index.html

```diff
@@ -1160,19 +1156,17 @@ <h3>Processing Model</h3>
 <a data-cite="HTML#or-equivalent" data-lt=
 'HTTP response codes equivalence'>equivalent</a>, then
 <ol>
-<li>If the current resource and the redirected resource are not
-from the <a data-cite="RFC6454#section-5">same origin</a> as the
-<a>current document</a>, and the <a>timing allow check</a>
-algorithm fails for either resource, set <a>redirectStart</a> and
-<a>redirectEnd</a> to 0. Then, return to step <a href=
-"#dfn-step-fetch-start">5</a> with the new resource.</li>
+<li>If the current resource fails the <a>timing allow check</a>
+algorithm, set <a>redirectStart</a> and <a>redirectEnd</a> to 0.
+Then, return to step <a href="#dfn-step-fetch-start">7</a> with the
+redirected resource.</li>
 <li>If the value of redirectStart is not set, let it be the value
 of fetchStart.</li>
 <li>Let redirectEnd be the value of responseEnd.</li>
 <li>Set all the attributes in the <a>PerformanceResourceTiming</a>
 object to 0 except <a>startTime</a>, <a>redirectStart</a>,
 <a>redirectEnd</a>, and <a>initiatorType</a>.</li>
-<li>Return to step <a href="#dfn-step-fetch-start">5</a> with the
+<li>Return to step <a href="#dfn-step-fetch-start">7</a> with the
```

Comment:
Can we add descriptive text instead of hard-coding a number?

Reply:
added